src/xenia/app/emulator_window.h

```diff
@@ -60,6 +60,8 @@ class EmulatorWindow {
   void UpdateTitle();
   void SetFullscreen(bool fullscreen);
   void ToggleFullscreen();
+  void TakeScreenshot();
+  void ExportImage(const xe::ui::RawImage& image);
```

Comment:
This looks like a generic function for saving images (or "exporting", though it's not clear what exactly "export" means there, "save", just like in that private function is a more descriptive word), but internally it calls all images "screenshot.png" even though it's not obvious from the name of the function that it should be used only for screenshots.

Reply:
I think my reasoning at the time was that if there was ever a feature implemented in the future to allow you to save screenshots in a different way (i.e through a dialog, maybe even choose different image formats) then you'd want to pass that logic through `ExportImage();`.

Exporting in this context simply means doing something with the image, might be saving it, might be converting it, etc. Either way, I think I'll rename it to `ExportScreenshot()` instead for clarity.